google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -205,7 +202,7 @@ public void run() {
     private final boolean isSingleUse;
     private final AtomicInteger asyncOperationsCount = new AtomicInteger();
 
-    private Object lock = new Object();
+    private final Object lock = new Object();
```

Comment:
The only change outside of syntax is this one, where we had to make this variable `final` to be used within a lambda.